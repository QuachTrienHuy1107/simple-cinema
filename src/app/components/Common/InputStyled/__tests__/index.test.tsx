import * as React from "react";
import { render } from "@testing-library/react";

import { InputStyled } from "..";

jest.mock("react-i18next", () => ({
    useTranslation: () => {
        return {
            t: str => str,
            i18n: {
                changeLanguage: () => new Promise(() => {}),
            },
        };
    },
}));

describe("<InputStyled  />", () => {
    it("should match snapshot", () => {
        const loadingIndicator = render(<InputStyled />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
