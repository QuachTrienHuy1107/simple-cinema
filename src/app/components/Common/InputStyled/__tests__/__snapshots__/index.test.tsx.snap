// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputStyled  /> should match snapshot 1`] = `
.c0 {
  border: 1px solid transparent;
  border-bottom: 1px solid #000;
  border-radius: 0;
  height: 44px;
  position: relative;
  font-size: 14px;
  width: 100%;
  padding: 0 20px;
}

.c0 .ant-input:focus {
  border: 1px solid transparent !important;
  border-bottom: 1px solid #000;
}

<div
  class=""
>
  <input
    class="ant-input ant-input-lg c0"
    type="text"
    value=""
  />
</div>
`;
